File: Python/PGS/LEVEL1/명예의 전당 (1)/gunwoo.py
def solution(k, score):
    answer = []
    for i in range(len(score)):
        while k > i:
            answer.append(min(score[0:i+1]))
            break
        if k <= i:
            scored = sorted(score[0:i+1], reverse = True)
            answer.append(scored[k-1])
    return answer

File: Python/PGS/LEVEL1/핸드폰 번호 가리기/gunwoo.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        for(int i=0; i<phone_number.length()-4; i++){
            answer += "*";
        }
        for(int i= phone_number.length()-4; i< phone_number.length(); i++){
            answer += phone_number.charAt(i); //문자열에 접근할 때는 charAt(i)
        }
        return answer;
    }
}